android/library/src/main/java/io/logto/android/Logto.kt

```diff
@@ -1,19 +1,82 @@
 package io.logto.android
 
+import android.app.Application
 import android.content.Context
 import io.logto.android.authflow.webview.WebViewAuthFlow
+import io.logto.android.callback.AuthenticationCallback
+import io.logto.android.config.LogtoConfig
 import io.logto.android.model.Credential
 import io.logto.android.storage.CredentialStorage
 
-class Logto {
-    companion object {
-        private var credentialStorage: CredentialStorage? = null
+class Logto private constructor() {
+
+    private lateinit var logtoConfig: LogtoConfig
+
+    private lateinit var application: Application
+
+    private var inited: Boolean = false
```

Comment:
this looks strange. inited should be checked by runtime instead of using a var. looks like `.isInitialized` is doing the job.